assets/js/modules/adsense/index.js

```diff
@@ -75,20 +75,17 @@ export const registerModule = ( modules ) => {
 			__( 'AdSense insights through Site Kit', 'google-site-kit' ),
 		],
 		checkRequirements: async ( registry ) => {
-			const adBlockerActive = await registry
+			const adBlockerActiveMsg = await registry
```

Comment:
Let's not use abbreviations for variable names like "msg". We can afford the bytes to write `adBlockerActiveMessage` instead 😉 

```js
			const adBlockerActiveMsg = await registry
```

Reply:
Alternatively: `message` is fine and then you can use it in the object return below 🙂